flink-state-backends/flink-statebackend-rocksdb/src/test/java/org/apache/flink/contrib/streaming/state/ttl/RocksDBTtlStateTestBase.java

```diff
@@ -41,6 +41,7 @@
 import static org.apache.flink.contrib.streaming.state.RocksDBOptions.TTL_COMPACT_FILTER_ENABLED;
 import static org.junit.Assert.assertEquals;
 import static org.junit.Assert.assertNotEquals;
+import static org.junit.Assert.assertTrue;
 
 /** Base test suite for rocksdb state TTL. */
 public abstract class RocksDBTtlStateTestBase extends TtlStateTestBase {
```

Comment:
I noticed the [CI](https://dev.azure.com/apache-flink/apache-flink/_build/results?buildId=2020&view=logs&j=0da23115-68bb-5dcd-192c-bd4c8adebde1&t=4ed44b66-cdd6-5dcf-5f6a-88b07dda665d) still broken.
There existed three `assertEquals("Original state should be cleared on access", ctx().emptyValue, ctx().getOriginal());` in `TtlStateTestBase.java` and I think they are both effected.